dotcom-rendering/src/web/components/ArticleBody.tsx

```diff
@@ -72,6 +72,20 @@ const globalLinkStyles = (palette: Palette) => css`
 	}
 `;
 
+const revealStyles = css`
+	/* We're using classnames here because we add and remove these classes
+	   using plain javascript */
+	.reveal {
+		animation: ${keyframes`
+			0% { opacity: 0; }
+			100% { opacity: 1; }
+		`} 4s ease-out;
```

Comment:
4s is quite a long animation!

Reply:
It's 5s on Frontend! I actually reduced it.

I suspect the reason for the length is because it can actually take a while for the content to be put in front of the reader. If they are half way down the page then the scroll takes a little time or if the content is being rendered that can also take a second or two.